doc/QUICKLOOK/index.rst

```diff
@@ -0,0 +1,8 @@
+=========
+quicklook
+=========
+
+.. toctree::
+   :maxdepth: 1
+
+   QUICKLOOK/index
```

Comment:
This is a circular link.  Please take a look at the existing index.rst files in the data model to see how they work.